include/realizations/catchment/LSTM_Realization.hpp

```diff
@@ -0,0 +1,156 @@
+#ifndef NGEN_LSTM_REALIZATION_HPP
+#define NGEN_LSTM_REALIZATION_HPP
+
+#include "Catchment_Formulation.hpp"
+#include <unordered_map>
+
+#ifdef NGEN_LSTM_TORCH_LIB_ACTIVE
```

Comment:
Hmm, if LSTM.h  isn't  available, then this entire class cannot  be properly constructed.  Perhaps wrapping  this entire header with `ifdef`  is  more appropriate?